cmdline/haskell-gi.hs

```diff
@@ -71,6 +71,20 @@ parseKeyValue s =
   let (a, '=':b) = break (=='=') s
    in (a, b)
 
+solveNameVersion::Overrides -> Text -> (Text, Maybe Text)
+solveNameVersion ovs name = (namewithoutversion, version)
+ where
```

Comment:
Perhaps better to use camelCase here for consistency, i.e. `namewithoutversion` -> `nameWithoutVersion`, for consistency.